lib/ShortcutTrigger/README.md

```diff
@@ -0,0 +1,20 @@
+# ShortcutTrigger
+
+A component which executes a function when a combination of keys are pressed at the same time.
+
+## Usage
+
+```
+<ShortcutTrigger shortcutKey="enter" onShortcutTrigger={func} withMetaKey />
+```
+
+### Props
+
+| Name                  | Type          | Default       | Required | Description                                         |
+| --------------------- |-------------- | ------------- | -------- |---------------------------------------------------- |
+| onShortcutTrigger    | Function      | ``      | Yes       | Executes when combination of keys are pressed.  |
+| shortcutKey   | string      | ``      | Yes       | Chosen key for the shortcut.  |
```

Comment:
I've put 'N/A' for props that have no default, we should agree on a standard approach for signifying this.

Reply:
Let's use 'N/A'. Updating now 👍